dpdd.tex

```diff
@@ -730,8 +730,13 @@ \subsubsection{Image Differencing Templates}
 \label{sec:templates}
 \dmreq{0280}
 
-Templates for difference image analysis will be created by coadding 6-months to a year long groups of visits. The coaddition process will take care to remove transient or fast moving objects (eg., asteroids) from the templates.
-Difference image analysis will use the appropriate template given the time of observation, airmass, and seeing.
+Coadded images will be used as templates for difference image analysis.
+The coaddition process will take care to remove transient or fast moving objects (eg., asteroids) from the templates.
+The time range of the epochs included in a template is TBD, and will be chosen to balance the detection of short-timescale transients (favoring shorter ranges) with the need to correct for DCR and maximize depth (favoring longer ranges).
```

Comment:
For AP the time range of coadded epochs doesn't influence the detection of short-timescale transients.  I would note that long-timescale coadds would begin to have problems for high proper-motion stars, however.

Reply:
> For AP the time range of coadded epochs doesn't influence the detection of short-timescale transients. I would note that long-timescale coadds would begin to have problems for high proper-motion stars, however.

Is it _just_ high proper-motion stars, then, that made us say we would only use 6-12 months of images in templates before?  If so, that seems strange, as I wouldn't have expected that to be a science case that cared about alerts (and whether it should care about diffim in DRP is at least non-obvious).  May be a question for @RobertLuptonTheGood or @ivezic .

> We have a requirement (DMS-REQ-0280) of templateMaxTimespan of 1yr. Are you suggesting we drop the upper limit as well?

I was unaware of that requirement, and while (as noted above) I don't want to say that we should drop it without first understanding why it's there, I think it may adversely affect the quality of the subtractions more broadly than whatever science case the req. was intended to support.  Noise (especially correlated noise), DCR, and artifact rejection will all push us towards wanting as many epochs as we can get in the templates.

Reply:
So this requirement on the templateMaxTimespan on the range of input exposures that can contribute to coadds was remove in LCR 2273, 2020-04-10, so this update is now consistent with the DMSR

Reply:
Sounds like we should remove "balance the detection of short-timescale transients (favoring shorter ranges)" from the text?  @ebellm ?

Reply:
We could say "minimize false positives due to high proper-motion stars (favoring shorter ranges)."

@TallJimbo, my interpretation of the original limit was not that folks wanted to use alerts for high PM star science, but that it would create false positives.